db/migrate/20181001131632_add_conversion_host_id_to_miq_request_tasks.rb

```diff
@@ -0,0 +1,45 @@
+class AddConversionHostIdToMiqRequestTasks < ActiveRecord::Migration[5.0]
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options, Hash
+    belongs_to :conversion_host, :class_name => "AddConversionHostIdToMiqRequestTasks::ConversionHost"
+  end
+
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    belongs_to :resource, :polymorphic => true
+  end
+
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :miq_request_tasks, :conversion_host_id, :bigint
+
+    MiqRequestTask.where(:type => 'ServiceTemplateTransformationPlanTask').each do |task|
+      host_id = task.options[:transformation_host_id]
+      next unless host_id
+      host = Host.find_by(:id => host_id)
+      next if host.nil?
+      task.conversion_host = ConversionHost.find_or_create_by!(:resource => host) do |ch|
+        ch.name                     = host.name
+        ch.vddk_transport_supported = true
+        ch.ssh_transport_supported  = false
+      end
+      task.options.delete(:transformation_host_id)
```

Comment:
Doing the delete here means that we will not remove this option from the existing hash if we can't find the host in the existing inventory. Is that the behavior we are expecting?

If it's okay to remove the host id from the hash either way, then you can replace the right hand side of the `host_id` assignment with this line.

Reply:
so if this `host` is gone from db, removing it from `options` seems better.

Reply:
@fdupont-redhat you can probably move it up [here](https://github.com/ManageIQ/manageiq-schema/pull/281/files#diff-c1d34c28716698230c0bdaeb06a6aeeeR21) and save a LOC